index.bs

```diff
@@ -2619,10 +2664,10 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
             :: |source|'s [=attribution source/source time=]
             : [=attribution rate-limit record/expiry time=]
             :: null
-            : [=attribution rate-limit record/event-level report ID=]
+            : [=attribution rate-limit record/entity ID=]
             :: null
```

Comment:
Is the `entity ID` field only `null` for fake event-level reports? If so, maybe add note that in the struct-field description for clarity.